### DISCUSSION TITLE
Moving Notification Options to the main options menu

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#16021] [Feature requests]
The notification options is a super useful tool that can prevent spam in your notification bar, but it's quite hidden away, and alot of people miss that this options menu even exists. I wanted to suggest moving the notifications (maybe a bell icon in the options menu) to the main options menu under a new tab. 
Mayhaps you could also add the option in there to switch between rct1 and rct2 style notification bars instead of having that be part of styling your gui? But that's another suggestion.

**Comment by Die4Ever** [comment]
also a separate notification channel for plugins would be cool, so you can disable every notification except the ones coming from plugins

**Reply by Broxzier** [reply]
This is a different feature request than what OP asks for.

**Comment by Broxzier** [comment]
I'm against this - the options window already lists so many items including ones that only affect one window. It makes much more sense to keep window-specific options to their own windows instead of clustering up the options window more.

**Comment by IntelOrca** [comment]
I don't think there is necessarily a right or wrong answer to this, the preference will vary from user to user. In this case though, it is simpler to stick with the status quo.
